R/spec.R

```diff
@@ -107,7 +107,7 @@ spec_validate_details <- function(call, signaller) {
     return(node_car(arg))
   }
 
-  fn <- as_string(node_car(call))
+  fn <- expr_deparse(node_car(call))
```

Comment:
I didn't test this since it's a pain without snapshots

Reply:
The testthat 3e PR is now merged if you would like to add snapshots